From a719e1f7e61ed8422f5a0c940e0f90851546c530 Mon Sep 17 00:00:00 2001
From: Arthur Liberman <arthur_liberman@hotmail.com>
Date: Fri, 3 May 2019 23:43:37 +0300
Subject: [PATCH] odroidn2/alsa: don't switch to default card on PCM signal Now
 that there is no more loopback from TDMC on the Odroid N2, this will correct
 a "no sound" situation for HDMI audio output with PCM.

---
 xbmc/cores/AudioEngine/Sinks/AESinkALSA.cpp | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/xbmc/cores/AudioEngine/Sinks/AESinkALSA.cpp b/xbmc/cores/AudioEngine/Sinks/AESinkALSA.cpp
index 4ff43b4b7c1..60de6b8d018 100644
--- a/xbmc/cores/AudioEngine/Sinks/AESinkALSA.cpp
+++ b/xbmc/cores/AudioEngine/Sinks/AESinkALSA.cpp
@@ -569,10 +569,6 @@ bool CAESinkALSA::Initialize(AEAudioFormat &format, std::string &device)
           break;
       }
     }
-    else
-    {
-      device = "@:CARD=AML" + GetAMLCardName(amlDeviceType) + ",DEV=0";
-    }
 
     aml_set_audio_passthrough(m_passthrough);
     SysfsUtils::SetInt("/sys/class/audiodsp/digital_codec", aml_digital_codec);
